RELEASING.md

```diff
@@ -82,6 +82,21 @@ This creates a new `release/99.0.0` branch to which various commits
 may be made to increment version numbers, update change logs, run
 any last test builds, etc.
 
+#### Verify Library Dependencies
+
+SimplyE uses several remote repositories as library dependencies published to Maven Central. Verify that each library dependency is referencing the latest version in the `build_libraries.gradle` file. The main libraries to focus on here are:  
```

Comment:
There will be some times when the latest version of a thing isn't necessarily what we want to release. I'd suggest something like "Verify that each library dependency is referencing the latest version in the `build_libraries.gradle` file or, if not, check with Product to see if exclusion is approved."